### PR TITLE
Added Some Details to Connector And Some Field Changes

### PR DESCRIPTION
**Pull request by Kinplemelon**

### CODE REVIEW

File: src/common/tools.ts [hunk 1]
```diff
@@ -106,3 +106,11 @@ export const getLabelFromValueInOptionList = <T>(
...
   return target?.label || ''
 }
+
+export const commonTimeUnits = [
+  { value: 'ms', label: 'ms' },
+  { value: 's', label: 's' },
+  { value: 'm', label: 'm' },
+  { value: 'h', label: 'h' },
+  { value: 'd', label: 'd' },
+]
```
Comment (ysfscream) [inline src/common/tools.ts:116]: Why not this?

```
export const commonTimeUnits = ['ms', 's', 'm', 'h', 'd']
```

Comment (ysfscream) [reply src/common/tools.ts:116]: I think it should be a problem of packaging in the component?

But I should provide options for both

`Label` and `Value` or `plain string arrays`, mainly to help in writing such data structures

File: src/i18n/RuleEngine.js [hunk 2]
```diff
@@ -327,4 +331,56 @@ export default {
...
+    zh: '留空则自动生成',
+    en: 'Empty will be automatically generated',
+  },
+  testTheConnection: {
+    zh: '测试连接',
+    en: 'Test the connection',
+  },
+  connectionSuccessful: {
+    zh: '连接成功',
+    en: 'Connection is successful',
```
Comment (ysfscream) [inline src/i18n/RuleEngine.js:384]: Connected?

File: src/views/RuleEngine/components/ConnectorDialog.vue [hunk 3]
```diff
@@ -5,152 +5,227 @@
...
-      }
-      submitLoading.value = false
-    }
+    : {},
+)
+const connectorTLS = ref(
+  props.edit && props.modelValue.ssl
+    ? _.cloneDeep(props.modelValue.ssl)
+    : _.cloneDeep(tlsParamsDefault),
+)
```
Comment (ysfscream) [inline src/views/RuleEngine/components/ConnectorDialog.vue:129]: Isn't it a bit complicated to include expressions as parameters here?

File: src/views/RuleEngine/components/ConnectorDialog.vue [hunk 4]
```diff
@@ -5,152 +5,227 @@
...
+    isTesting.value = false
+  }
+}
+
+const submitConnector = async (isEdit) => {
+  let res
+  submitLoading.value = true
+  const data = getConnectorData()
+  if (isEdit) {
+    const id = connectorData.value.id
```
Comment (ysfscream) [inline src/views/RuleEngine/components/ConnectorDialog.vue:171]: const { id } = connectorData.value

Better? Just a suggestion

**Review by ysfscream — APPROVED** [review]

File: src/i18n/RuleEngine.js (hunk 2 above)
Comment (Kinplemelon) [reply src/i18n/RuleEngine.js:384]: When I search in English, 'Connection successful' is more; but my grammar is poor, if you are sure the 'connected successful' is right, I will change it.

File: src/views/RuleEngine/components/ConnectorDialog.vue (hunk 4 above)
Comment (Kinplemelon) [reply src/views/RuleEngine/components/ConnectorDialog.vue:171]: yes

File: src/views/RuleEngine/components/ConnectorDialog.vue (hunk 3 above)
Comment (Kinplemelon) [reply src/views/RuleEngine/components/ConnectorDialog.vue:129]: Actually, this is historical code.. But when I want to try to modify it, I found a bug in this logic, tks

Comment (Kinplemelon) [reply src/views/RuleEngine/components/ConnectorDialog.vue:129]: No bug, just need to optimize the logic 😵

File: src/i18n/RuleEngine.js (hunk 2 above)
Comment (ysfscream) [reply src/i18n/RuleEngine.js:384]: No, I meant "Connected". I can confirm that this is a better indication that the connection was successful.

Comment (Kinplemelon) [reply src/i18n/RuleEngine.js:384]: This msg is pop up after Test The Connection,  is it right to use 'connected'?

**Pull request merged**